litmus-portal/frontend/src/pages/WorkflowDetails/index.tsx

```diff
@@ -154,16 +151,15 @@ const WorkflowDetails: React.FC = () => {
                   nodes={
                     (JSON.parse(workflow.execution_data) as ExecutionData).nodes
                   }
+                  onClick={() => setIsInfoToggled(true)}
```

Comment:
There is a possibility of a re-render of the graph on click....so check that. It might have happened before, I'm not entirely sure